src/demos/shell/block-configurations.html

```diff
@@ -30,7 +30,12 @@
     </style>
 
     <script>
-      require(["esri/WebMap", "esri/views/MapView", "esri/widgets/LayerList","esri/widgets/Zoom"], function(WebMap, MapView, LayerList, Zoom) {
+      require(["esri/WebMap", "esri/views/MapView", "esri/widgets/LayerList", "esri/widgets/Zoom"], function (
```

Comment:
Are these bonus changes? 🎉  Just asking since it doesn't have any row-related changes.

Reply:
I think that might have happened due to a formatter.

Reply:
Gotcha. Thanks for checking. 😄